lib/MusicBrainz/Server/Controller/Root.pm

```diff
@@ -225,7 +225,10 @@ sub begin : Private
     try {
         my $store = $c->model('MB')->context->store;
 
-        if ($c->user_exists) {
+        if ($c->user_exists && !(
+                $c->stash->{server_details}->{staging_server} &&
+                $c->stash->{server_details}->{is_sanitized}))
+        {
```

Comment:
This also disables the notes-received functionality, but we should keep that part enabled.

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/1552/commits/2d8bdb6f04cd054f3f066f3873e6527b4457c73a